packages/perf-benchmarks/package.json

```diff
@@ -6,10 +6,10 @@
         "start": "best"
     },
     "devDependencies": {
-        "@best/cli": "4.0.0-beta5",
-        "@best/runner-headless": "4.0.0-beta5",
-        "@best/runner-remote": "4.0.0-beta5",
-        "@best/store-aws": "4.0.0-beta5",
+        "@best/cli": "4.0.0-beta6",
```

Comment:
`beta6` has a security certificate issue so we should wait for `beta7` /cc @diervo